src/main/java/pageobjectsnew/IndexPageNew.java

```diff
@@ -0,0 +1,274 @@
+    public WebElement cartItemsNumber;
+
+    @FindBy(xpath = "//*[contains(@class,'bestsellers')]/div/div/div/ul/li/div")
+    public List<WebElement> bestsellersItems;
+
+    @FindBy(xpath = "//*[contains(@class,'featured')]/div/div/div/ul/li/div")
+    public List<WebElement> featuredItems;
+
+    @FindBy(css = "li[class*=logoff]")
+    public WebElement logOffCss;
+
+    @FindBy(css = "li.tab.admin a")
+    public WebElement adminLoginLinkCss;
+
+    public IndexPageNew clickOnGoToAdminPanelLink() {
+        clickOnElement(adminLoginLinkCss, "Admin login link", driver);
+        return this;
+    }
+
+    public IndexPageNew clickOnLoginButton() {
+        waitForElementVisible(loginButtonCss, driver);
+        clickOnElement(loginButtonCss, "Login Button", driver);
+        return this;
+    }
+
+    public IndexPageNew openSite() {
+        //get index page
+        getURL(Constants.URL, driver);
+        return this;
+    }
+
+    public IndexPageNew clickOnCreateAccountLink() {
+        waitForElementVisible(createAccountLinkCss, driver);
+        clickOnElement(createAccountLinkCss, "Create account link", driver);
+        return this;
+    }
+
+    public String getPageTitleText(){
+        return pageTitle.getText();
+    }
+
+    public String getErrorText(){
+        return loginErrorMessage.get(0).getText();
+    }
+
+    public IndexPageNew login(User user, String method){
+        sendKeysClear(email, "Email form", user.getEmail(), driver);
+        sendKeysClear(password, "Password form", user.getPassword(), driver);
+
+        switch (method){
+            case "button":
+                clickOnSignInButton();
+                break;
+            case "enter":
+                password.sendKeys(Keys.ENTER);
+                break;
+            default:
+                clickOnSignInButton();
+                break;
+        }
+        return this;
+    }
+
+    public IndexPageNew createAccount(User user){
+        waitForElementVisible(createAccountButtonCss, driver);
+
+        sendKeysClear(emailCreate, "Email form create user", user.getEmail(), driver);
+
+        sendKeysClear(passwordCreate, "Password form create user", user.getPassword(), driver);
+
+        sendKeysClear(passwordConfirmCreate, "Second password form create user", user.getPassword(), driver);
+
+        clickOnCreateAccountButton();
+
+        return this;
+    }
+
+    public IndexPageNew clickOnSignInButton() {
+        waitForElementVisible(signInButtonCss, driver);
+        clickOnElement(signInButtonCss, "SignIn button", driver);
+        return this;
+    }
+
+    public IndexPageNew clickOnCreateAccountButton() {
+        waitForElementVisible(createAccountButtonCss, driver);
+        clickOnElement(createAccountButtonCss, "Create account button", driver);
+        return this;
+    }
+
+    public IndexPageNew addRandomItemsFromIndexPageToCart(int item) {
+
+        ItemPage itemPage = initElements(driver, ItemPage.class);
```

Comment:
It looks like addRandomItemsFromIndexPageToCartmethod should be in ItemPage class.